docx4j-core/src/main/java/org/docx4j/utils/StringUtils.java

```diff
@@ -0,0 +1,27 @@
+package org.docx4j.utils;
+
+public class StringUtils {
+
+    /**
+     * Escape user input to avoid directory traversal attacks.
+     *
+     * @param userInput
+     * @return
+     */
+    public static String escapeUserInput(String userInput) {
+        return userInput.replaceAll("[^a-zA-Z0-9.-]", "_");
```

Comment:
is this inclusive enough?

Reply:
i guess this is just going to be whatever we send over so it's probably fine

Reply:
i'll use a more specific name, and I think there's also a teeny bug in there, i only scoped it to acceptable file name characters, so it should probably be named `escapeFilepath` or something. then if someone enters "/path/to/file.txt" the slashes will get replaced so i think i need to add `//` and `\/` to the regex